examples/first-component/src/types/Todo.ts

```diff
@@ -0,0 +1,5 @@
+export interface Todo {
+  id: number;
+  text: string;
```

Comment:
this is okay but need to naming more meaning full instead of just "text"

Reply:
Fixed